src/EnergyPlus/HVACManager.cc

```diff
@@ -2397,7 +2397,7 @@ void ReportInfiltrations(EnergyPlusData &state)
         } else if (thisZoneHB.MAT <= thisZone.OutDryBulbTemp) {
 
             thisInfiltration.InfilHeatGain =
-                thisInfiltration.MCpI_temp * (thisZone.OutDryBulbTemp - thisZoneHB.MAT) * TimeStepSys * ADSCorrectionFactor;
+                thisInfiltration.MCpI_temp * (thisZone.OutDryBulbTemp - thisZoneHB.MAT) * TimeStepSysSec * ADSCorrectionFactor;
```

Comment:
If you look at the `if` statement above, or the [commit where the bug was introduced](https://github.com/NREL/EnergyPlus/commit/bc5300d04e509fa40a25c6f0bc306755f657fa8f), it should be pretty obvious that this is a correct change.

Reply:
Yeah, this is an obvious change both based on the context and the original commit that changed it.  Thanks.